src/test/java/nurseybook/testutil/TypicalTasks.java

```diff
@@ -1,89 +1,90 @@
-package seedu.address.testutil;
-
-import static seedu.address.logic.commands.TaskCommandTestUtil.VALID_DATE_JAN;
-import static seedu.address.logic.commands.TaskCommandTestUtil.VALID_DATE_NOV;
-import static seedu.address.logic.commands.TaskCommandTestUtil.VALID_DESC_COVID;
-import static seedu.address.logic.commands.TaskCommandTestUtil.VALID_DESC_MEDICINE;
-import static seedu.address.logic.commands.TaskCommandTestUtil.VALID_DESC_PAPERWORK;
-import static seedu.address.logic.commands.TaskCommandTestUtil.VALID_DESC_VACCINE;
-import static seedu.address.logic.commands.TaskCommandTestUtil.VALID_NAME_ALEX;
-import static seedu.address.logic.commands.TaskCommandTestUtil.VALID_NAME_KEITH;
-import static seedu.address.logic.commands.TaskCommandTestUtil.VALID_TIME_SEVENPM;
-import static seedu.address.logic.commands.TaskCommandTestUtil.VALID_TIME_TENAM;
+package nurseybook.testutil;
+
+import static nurseybook.logic.commands.TaskCommandTestUtil.VALID_DATE_JAN;
+import static nurseybook.logic.commands.TaskCommandTestUtil.VALID_DATE_NOV;
+import static nurseybook.logic.commands.TaskCommandTestUtil.VALID_DESC_COVID;
+import static nurseybook.logic.commands.TaskCommandTestUtil.VALID_DESC_MEDICINE;
+import static nurseybook.logic.commands.TaskCommandTestUtil.VALID_DESC_PAPERWORK;
+import static nurseybook.logic.commands.TaskCommandTestUtil.VALID_DESC_VACCINE;
+import static nurseybook.logic.commands.TaskCommandTestUtil.VALID_NAME_ALEX;
+import static nurseybook.logic.commands.TaskCommandTestUtil.VALID_NAME_KEITH;
+import static nurseybook.logic.commands.TaskCommandTestUtil.VALID_TIME_SEVENPM;
+import static nurseybook.logic.commands.TaskCommandTestUtil.VALID_TIME_TENAM;
 
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
 
-import seedu.address.model.AddressBook;
-import seedu.address.model.task.Recurrence.RecurrenceType;
-import seedu.address.model.task.Task;
+import nurseybook.model.NurseyBook;
+import nurseybook.model.task.Recurrence;
+import nurseybook.model.task.Task;
 
 public class TypicalTasks {
 
     public static final Task KEITH_INSULIN = new TaskBuilder().withDesc(VALID_DESC_MEDICINE)
             .withDateTime(VALID_DATE_NOV, VALID_TIME_SEVENPM).withNames(VALID_NAME_KEITH)
             .withStatus("false", "true")
-            .withRecurrence(RecurrenceType.NONE.name())
+            .withRecurrence(Recurrence.RecurrenceType.NONE.name())
```

Comment:
Missing imports in this file as well that would be better to include at the start!